_data/iot.yml

```diff
@@ -14,6 +14,9 @@ websites:
     twitter: ecobee
     facebook: ecobee
```

Comment:
```suggestion
```

Reply:
Yes, but they are not used/needed if some tfa method is present (see the failed Jekyll check).

Reply:
Aaaah gotcha.... That makes sense.  Sorry... lack of coffee fail!